client.js

```diff
@@ -45,100 +42,338 @@ bot.on('message', function (message) {
+                                                // Snooze this watch for 6 hours
+                                                db.snooze('watch', watch.id);
+                                                user.send(`Sleep is good.  Pausing notifications for the next 6 hours on your \`\`${item}\`\` watch on \`\`${server}\`\`.  Click 💤 again to unsnooze.  To snooze all watches, use \`\`!snooze\`\``).catch(console.error);
+                                                break;
+                                            case '❌':
+                                                // Delete this watch
+                                                db.endWatch(null, null, null, watch.id);
+                                                user.send(`Very well, no longer watching for auctions of \`\`${item}\`\`\ on \`\`${server}\`\`.`);
+                                                break;
+                                            case '♻': //extend watch
+                                                db.extendWatch(watch.id)
+                                                user.send(`Good things come to those who wait.  I added another 7 days to your \`\`${item}\`\` watch on \`\`${server}\`\`.`);
+                                                break;
+                                            default:
+                                                break;
+                                        }
+                                    })
+                                    collector.on('remove', (reaction, user) => {
+                                        switch (reaction.emoji.name) {
+                                            case '💤':
+                                                // unsnooze watch
+                                                db.unsnooze('watch', watch.id);
+                                                user.send(`Rise and grind.  No longer snoozing on your \`\`${item}\`\` watch on \`\`${server}\`\`.`).catch(console.error);
+                                                break;
+                                            case '❌':
+                                                // Renew watch
+                                                db.addWatch(null, null, null, null, watch.id);
+                                                user.send(`Ok, watching for auctions of \`\`${item}\`\` on P1999 \`\`${server}\`\` again.`);
+                                                break;
+                                            case '🔕':
+                                                //unblock seller's auctions for this watch
+                                                db.unblockSeller(user.id, seller, watch.server, watch.id)
+                                                user.send(`Let's cut out the noise!  No longer notifying you about auctions from ${seller} with regard to this watch.\n  To block ${seller} on all present and future watches, use \`\`!add block: ${seller}`);
```

Comment:
Also, isn't this totally the wrong message for "unblock"?

Reply:
Yup.